src/apis/geolocation.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+export default async (city, stateCode, countryCode) => {
+  try {
+    let coords = [];
+    const response = await axios.get(
+      `http://api.openweathermap.org/geo/1.0/direct?q=${city}
```

Comment:
It needs to be HTTPS protocol. It's throwing an error.

```suggestion
      `https://api.openweathermap.org/geo/1.0/direct?q=${city}
```

Reply:
Why is that not throwing an error to me but it does to you?

Reply:
Your code may break before calling the api